### DISCUSSION TITLE
archives/671/

### DISCUSSION THREAD

**Discussion by giscus** [songtianlun/songtianlun.github.io#5] [Announcements]
# archives/671/

ChatGPT 是一个训练有素的语言模型，由 OpenAI 公司训练而成。目的是帮助回答问题，提供信息和支持，但不具备人类的感知能力，也不会回答所有问题。 以上自我介绍原

https://www.frytea.com/archives/671/

**Comment by TomAngsChen** [comment]
注册一直是一个问题，想注册一些站点或者app，总是会接到一些垃圾短信和一些骚扰电话
不过，办法总比困难多，我发现了一个比较好用的接码平台，妈妈再也不用担心我被打扰了
站点分享给大家：www.sms-999.com

**Comment by aichatgptai** [comment]
感谢博主分享，已经注册成功可以看博主这篇文章：https://share.weiyun.com/0mtW0MKF
